src/System.Console/src/System/ConsolePal.WinRT.cs

```diff
@@ -0,0 +1,256 @@
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public static int LargestWindowHeight
+        {
+            get { throw new PlatformNotSupportedException(); }
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public static int WindowLeft
+        {
+            get { return 0; }
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public static int WindowTop
+        {
+            get { return 0; }
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public static int WindowWidth
+        {
+            get { throw new PlatformNotSupportedException(); }
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public static int WindowHeight
+        {
+            get { throw new PlatformNotSupportedException(); }
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public static void SetWindowPosition(int left, int top) { throw new PlatformNotSupportedException(); }
+
+        public static void SetWindowSize(int width, int height) { throw new PlatformNotSupportedException(); }
+
+        public static bool CursorVisible
+        {
+            get { throw new PlatformNotSupportedException(); }
+            set
+            {
+                if (!Console.IsOutputRedirected)
+                    throw new PlatformNotSupportedException();
+            }
+        }
+
+        public static int CursorLeft
+        {
+            get
+            {
+                if (!Console.IsOutputRedirected && !Console.IsInputRedirected)
+                    throw new PlatformNotSupportedException();
+                return 0;
+            }
+        }
+
+        public static int CursorTop
+        {
+            get
+            {
+                if (!Console.IsOutputRedirected && !Console.IsInputRedirected)
+                    throw new PlatformNotSupportedException();
+                return 0;
+            }
+        }
+
+        public static void MoveBufferArea(int sourceLeft, int sourceTop, int sourceWidth, int sourceHeight, int targetLeft, int targetTop) { throw new PlatformNotSupportedException(); }
+
+        public static void MoveBufferArea(int sourceLeft, int sourceTop, int sourceWidth, int sourceHeight, int targetLeft, int targetTop, char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor) { throw new PlatformNotSupportedException(); }
+
+        public static bool IsInputRedirectedCore() { throw new PlatformNotSupportedException(); }
+
+        public static bool IsOutputRedirectedCore() { throw new PlatformNotSupportedException(); }
+
+        public static bool IsErrorRedirectedCore() { throw new PlatformNotSupportedException(); }
+
+        public static void SetConsoleInputEncoding(Encoding enc) { }
+
+        public static void SetConsoleOutputEncoding(Encoding enc) { }
```

Comment:
This is the same case for the methods where I return default values (e.g. WindowLeft_get returns 0).